gobblin-modules/gobblin-azkaban/src/main/java/org/apache/gobblin/service/modules/orchestration/AzkabanClient.java

```diff
@@ -130,10 +138,12 @@ private void initializeClient() throws AzkabanClientException {
 
   private void initializeSessionManager() throws AzkabanClientException {
     if (sessionManager == null) {
-      this.sessionManager = new AzkabanSessionManager(this.httpClient,
-                                                      this.url,
-                                                      this.username,
-                                                      this.password);
+      try {
+        this.sessionManager = new AzkabanSessionManager(this.httpClient, this.url, this.username, this.password);
+      }
+      catch(Exception e) {
+        log.error("Failed to initialize session manager due to: ", e);
```

Comment:
I don't see the SessionManager actually throwing anything in the constructor so the try/catch is redundant